classes/World.py

```diff
@@ -229,3 +233,12 @@ def get_scooters_on_trip(self) -> [(int, int, int)]:
     def get_discount(self):
         # Divide by 60 as there is 60 minutes in an hour. We want this number in hours to avoid big numbers is the power
         return DISCOUNT_RATE ** (self.time / 60)
+
+    def get_filename(self):
+        return (
+            f"{self.created_at}_World_T_e{self.time}_t_{self.shift_duration}_"
+            f"S_c{len(self.state.clusters)}_s{len(self.state.get_scooters())}.pickle"
+        )
```

Comment:
Maybe add a field that represents how many training rounds the state/value function has been trained?

Reply:
We have no field for this yet. We can add it when we start serious training